gef-ui/server/services/api.js

```diff
@@ -60,10 +60,44 @@ const getFacilities = async (applicationId) => {
   }
 };
 
+const getAutomaticCover = async () => {
```

Comment:
Is this temporary? Will it call an api to get the automatic cover

Reply:
Yeah it was temporary for the demo. I am currently updating it as we speak.